lms/static/sass/course/courseware/_sidebar.scss

```diff
@@ -1,234 +1,168 @@
-      background-color: $sidebar-chapter-bg;
-      box-shadow: 0 1px 0 $white inset, 0 -1px 0 $shadow-l1 inset;
-      @include transition(background-color .1s linear 0s);
-
-      &.is-open {
-        background: $white;
-      }
-
-      &:first-child {
-        border-radius: 3px 0 0 0;
-      }
-
-      &:last-child {
-        border-radius: 0 0 0 3px;
-        box-shadow: 0 1px 0 $white inset;
-      }
-
-      &:hover, &:focus {
-        background-color: $white;
-      }
+        font-weight: normal;
     }
 
-    ul.ui-accordion-content {
-      background: transparent;
-      border: none;
-      border-radius: 0;
-      margin: 0;
-      padding: 9px 0 9px 9px;
-      overflow: auto;
-      width: 100%;
-
-      li {
-        border-bottom: 0;
-        border-radius: 0;
-        margin-bottom: ($baseline/5);
-
-        a {
-          background: transparent;
-          border-radius: 4px;
-          display: block;
-          @include padding( ($baseline/4), ($baseline*1.5), ($baseline/4), ($baseline/2));
-          position: relative;
-          text-decoration: none;
-
-          p {
-            font-weight: bold;
-            font-family: $sans-serif;
-            margin-bottom: 0;
-            line-height: 1.3;
-
-            &.subtitle {
-              @extend %t-copy-sub2;
-              @extend %t-weight2;
-              display: inline-block;
-              width: 90%;
-              color: $gray-d1;
-              margin: 0;
-
-              &:empty {
-                display: none;
-              }
-
-              // definitions for proctored exam attempt status indicators
-              i.verified {
-                color: $success-color;
-              }
-
-              i.rejected {
-                color: $alert-color;
-              }
-
-              i.error {
-                color: $alert-color;
-              }
-            }
-          }
-
-          &:hover, &:focus {
-            background: $shadow-l1;
-
-            > a p {
-              color: $gray-d3;
-            }
-          }
-
-          &:active {
-            box-shadow: inset 0 1px 14px 0 $shadow-l1;
-
-            &:after {
-              opacity: 1.0;
-              right: 15px;
-            }
-          }
-        }
-
-        &.active {
-          @extend %t-weight5;
-
-          &:after {
-            content: '›';
-            position: absolute;
-            top: 50%;
-            right: 20px;
-            margin-top: -13px;
-            font-size: 30px;
-            font-weight: normal;
-            color: #333;
-            opacity: 0;
-            @include transition(none);
-          }
-
-          > a {
-            border: 1px solid $border-color-1;
-            box-shadow: 0 1px 0 rgba(255, 255, 255, .35) inset;
-            background: $sidebar-active-image;
-
-            &:after {
-              opacity: 1.0;
-              right: 15px;
-            }
-
-            p {
-              color: #333;
+    #accordion {
```

Comment:
I know you're carrying this over from existing styles, but using the id attr selector here is not our norm. Is it too much trouble to change this to a class-based style selector (keeping the id or even a separate behavioral class, @AlasdairSwan's recent ideas, for any JS you need)?